advanced/Templates/pihole-FTL.service

```
@@ -60,16 +89,16 @@ stop() {
 
     if is_running; then
       echo "Not stopped; may still be shutting down or shutdown may have failed, killing now"
-      pkill -xo -9 "pihole-FTL"
-      exit 1
+      pkill -9 -F "${FTL_PID_FILE}"
```

Use more generic/low level `kill` command:
```suggestion
      kill -9 "$FTL_PID"
```